src/components/link/notes.stories.mdx

```diff
@@ -1,14 +1,10 @@
----
-menu: Components
-name: Link
-route: /components/link
----
-
-import { Playground, Props } from "docz"
+import { Meta, Preview, Props, Story } from "@storybook/addon-docs/blocks"
 
 import Link from "./"
 import { ExternalLink, InternalLink } from "./"
 
+<Meta title="Components/Link" component={Link} />
```

Comment:
Looks like we lost the usage code here. [See here](https://deploy-preview-27--gatsby-theme-ample-components.netlify.com/?path=/story/components-link--page#usage).